apps/website-25/src/components/courses/exercises/FreeTextResponse.tsx

```diff
@@ -65,17 +67,29 @@ const FreeTextResponse: React.FC<FreeTextResponseProps> = ({
           className={`free-text-response__textarea p-4${
             (!isEditing && exerciseResponse) ? ' free-text-response__textarea--saved container-active bg-[#63C96533] border-[#63C965] text-[#2A5D2A]' : ' container-lined'
           }`}
-          placeholder="Enter your answer here"
+          placeholder={isLoggedIn ? 'Enter your answer here' : 'Login to save your answers'}
           onChange={() => setIsEditing(true)}
+          disabled={!isLoggedIn}
         />
       </div>
-      <CTALinkOrButton
-        className="free-text-response__submit"
-        variant="primary"
-        type="submit"
-      >
-        Save
-      </CTALinkOrButton>
+      {isLoggedIn ? (
+        <CTALinkOrButton
+          className="free-text-response__submit"
+          variant="primary"
+          type="submit"
+        >
+          Save
+        </CTALinkOrButton>
+      ) : (
+        <CTALinkOrButton
+          className="free-text-response__login-cta"
+          variant="primary"
+          url="https://course.bluedot.org/login"
```

Comment:
Change this to the current domain, not course.bluedot.org (which is old course hub).

consider: You also might want to pass in a redirect_to parameter with the current url, so they end up back here. i.e.

```ts
url={`/login?redirect_to=${encodeUriComponent(window.location.href)}`}
```